### ISSUE TITLE
Option to use java-time instead of joda

### ISSUE THREAD

**Issue by conan** [mcohen01/amazonica#411]
Currently Amazonica [converts](https://github.com/mcohen01/amazonica#conversion-of-returned-types) `java.util.Date`s to `org.joda.time.DateTime`s.  Now that we have the time API from Java 8, and [good support](https://github.com/dm3/clojure.java-time) for it in clojure, it would be great to have an option to work with the native classes, or at least to leave `java.util.Date`s as they are without conversion.

**Comment by henryw374** [comment]
I think just leave dates alone and/or make the conversion extensible. 

interested in a PR? if so, is a breaking change (ie no joda dep) ok?

**Comment by henryw374** [comment]
PR here https://github.com/mcohen01/amazonica/pull/431 

In the meantime, it's quite straightforward to add one extend-protocol of IMarshall and one alter-var-root of to-date in your own project.

**Comment by ieugen** [comment]
I also support moving to standard libraries and dropping extra code.
Long term is easier for maintenance, including security related issues (post log4j bulnerabilities).
Less is more.

**Comment by henryw374** [comment]
yeah I actually stopped using Amazonica and use the official java v2 client directly - not difficult from Clojure & it's well documented

**Comment by Limess** [comment]
This'd be very nice to stop us from having to pull in `clj-time` in all our libraries. We'd prefer to not immediately have to switch to the java SDK or https://github.com/cognitect-labs/aws-api.
